03/app.js

```diff
@@ -66,14 +66,14 @@ books.getTranslator = function(isbn, lang) {
         return null;
     }
 
-    if (translator[lang] === null) {
+    if (translator[lang] === null || translator[lang] === undefined) {
```

Comment:
Tutaj bym uważał. W zadaniu jest jasno napisane, że `null` więc tylko bym sprawdzał tą wartość.
`undefined` to co innego - wtedy ma być zwracane `null` (ostatni warunek, który masz)